src/ln/channel.rs

```diff
@@ -357,8 +357,8 @@ const UNCONF_THRESHOLD: u32 = 6;
 const BREAKDOWN_TIMEOUT: u16 = 6 * 24 * 7; //TODO?
 /// The amount of time we're willing to wait to claim money back to us
 const MAX_LOCAL_BREAKDOWN_TIMEOUT: u16 = 6 * 24 * 14;
-const COMMITMENT_TX_BASE_WEIGHT: u64 = 724;
-const COMMITMENT_TX_WEIGHT_PER_HTLC: u64 = 172;
+pub const COMMITMENT_TX_BASE_WEIGHT: u64 = 724;
```

Comment:
May want to add a comment noting these are just exposed for channelmanager tests.

Reply:
Done.